apps/studio/components/interfaces/Home/Connect/Connect.tsx

```diff
@@ -0,0 +1,324 @@
+    }
+  }
+
+  function handleConnectionType(type: string) {
+    if (type === 'frameworks') {
+      setConnectionObject(FRAMEWORKS)
+      handleConnectionTypeChange(FRAMEWORKS)
+    }
+
+    if (type === 'orms') {
+      setConnectionObject(ORMS)
+      handleConnectionTypeChange(ORMS)
+    }
+  }
+
+  const getChildOptions = () => {
+    const parent = connectionObject.find((item) => item.key === selectedParent)
+    if (parent && parent.children.length > 0) {
+      return parent.children
+    }
+    return []
+  }
+
+  const getGrandchildrenOptions = () => {
+    const parent = connectionObject.find((item) => item.key === selectedParent)
+    const subCategory = parent?.children.find((child) => child.key === selectedChild)
+    if (subCategory && subCategory.children.length > 0) {
+      return subCategory.children
+    }
+    return []
+  }
+
+  const { data: projectSettings } = useProjectSettingsQuery({ projectRef })
+
+  const { data: apiSettings } = useProjectApiQuery({
+    projectRef,
+  })
+
+  // Get the API service
+  const apiService = (projectSettings?.services ?? []).find(
+    (x: any) => x.app.id == DEFAULT_PROJECT_API_SERVICE_ID
+  )
+
+  const canReadAPIKeys = useCheckPermissions(PermissionAction.READ, 'service_api_keys')
+
+  // Get the API service
+  const apiHost = canReadAPIKeys
+    ? `${apiSettings?.autoApiService?.protocol ?? 'https'}://${
+        apiSettings?.autoApiService?.endpoint ?? '-'
+      }`
+    : ''
+  const apiUrl = canReadAPIKeys ? apiHost : null
+
+  const anonKey = canReadAPIKeys
+    ? apiService?.service_api_keys.find((key) => key.tags === 'anon')?.api_key ?? null
+    : null
+
+  const projectKeys = { apiUrl, anonKey }
+
+  const filePath = getContentFilePath({
+    connectionObject,
+    selectedParent,
+    selectedChild,
+    selectedGrandchild,
+  })
+
+  return (
+    <>
+      <div>
+        <Dialog_Shadcn_ open={true}>
+          <DialogTrigger_Shadcn_ asChild>
+            <Button type="default">
+              <span className="flex items-center gap-2">
+                <Plug size={12} className="rotate-90" /> <span>Connect</span>
+              </span>
+            </Button>
+          </DialogTrigger_Shadcn_>
+          <DialogContent_Shadcn_ className={cn('sm:max-w-5xl p-0')}>
+            <DialogHeader_Shadcn_ className="pb-0">
+              <DialogTitle_Shadcn_>Connect to your project</DialogTitle_Shadcn_>
+              <DialogDescription_Shadcn_>
+                Get the connection strings and environment variables for your app
+              </DialogDescription_Shadcn_>
+            </DialogHeader_Shadcn_>
+
+            <Tabs_Shadcn_
+              defaultValue="direct"
+              onValueChange={(value) => handleConnectionType(value)}
+            >
+              <TabsList_Shadcn_ className={cn('flex gap-4', DIALOG_PADDING_X_Shadcn_)}>
+                <TabsTrigger_Shadcn_ key="direct" value="direct" className="px-0">
+                  Direct Connection
```

Comment:
let's avoid the wording direct connection - we want to advocate using the pooler over directly connecting too
```suggestion
                  Connection String
```